libs/dh/core/shell/src/lib/dh-core-shell.module.ts

```diff
@@ -14,31 +14,48 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { HttpClientModule } from '@angular/common/http';
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+
 import { DhTranslocoModule } from '@energinet-datahub/dh/globalization/configuration-localization';
 
-import { ShellComponent } from './shell/shell.component';
-import { ShellModule } from './shell/shell.module';
+import {
+  DhCoreShellComponent,
+  DhCoreShellComponentScam,
+} from './dh-core-shell.component';
 
 const routes: Routes = [
   {
     path: '',
-    component: ShellComponent,
+    component: DhCoreShellComponent,
     children: [
-      // Lazy feature routes
+      {
+        path: '',
+        redirectTo: 'metering-point',
+        pathMatch: 'full',
+      },
+      {
+        path: 'metering-point',
+        loadChildren: () =>
+          import('@energinet-datahub/dh/metering-point/shell').then(
+            (esModule) => esModule.DhMeteringPointShellModule
+          ),
+      },
     ],
   },
+  // { path: '**', component: PageNotFoundComponent },
 ];
 
 @NgModule({
   exports: [RouterModule],
   imports: [
-    HttpClientModule,
```

Comment:
issue(non-blocking): `HttpClientModule` must only be imported exactly once and in the root injector only. We need this for DataHub API data access which I'm currently implementing in a separate branch so let's keep it.

issue(blocking): I see that `HttpClientModule` is currently being imported in `WattIconModule`. This will lead to errors because multiple instances of HTTP-related dependencies will be instantiated.

Reply:
`HttpClientModule` has now been removed from `WattIconModule`

Reply:
issue(blocking): we're still missing an `HttpClientModule` import in this core shell module for the Watt Icon to work